library/blivet.py

```diff
@@ -611,6 +611,11 @@ def _create(self):
         if safe_mode:
             raise BlivetAnsibleError("cannot create new RAID in safe mode")
```

Comment:
I can't comment on the change itself, but this line above does not look right. Does it mean that every RAID creation is considered unsafe and users must set safe_mode off for it to be allowed, even if the RAID is to be created on an empty disk set?

Reply:
Fixed. Thanks for noticing.

Reply:
We must not fall into the opposite problem of allowing to destroy existing data in safe mode though. See #168 .